app/routes/__frontend/resources/virtual-coffee-handbook/guides-to-virtual-coffee/hacktoberfest-initiative.mdx

```diff
@@ -0,0 +1,48 @@
+---
+meta:
+  title: Hacktoberfest Initiative
+  description: An event where we encourage and support our members in contributing to and maintaining open source during Hacktoberfest.
+hero:
+  Hero: UndrawPumpkin
+tags: [memberresources]
+order: 8
+---
+
+import LeadText from '~/components/content/LeadText';
+import TextContainer from '~/components/content/TextContainer';
+
+<TextContainer background="light" showBackToTopLink={false}>
+
+<LeadText>
+
+Do you know that Virtual Coffee is also open source? That's one of the reasons we participate in [DigitalOcean's Hacktoberfest](https://hacktoberfest.digitalocean.com) through our Hacktoberfest Initiative!
+
+</LeadText>
+
+</TextContainer>
+
+## Table of Contents
+
+<TextContainer>
+
+---
+
+## What Is Hacktoberfest?
+
+Hacktoberfest is a month-long virtual event that encourages and supports open source contributions. Open Source Software (OSS) is a codebase that the public can view, contribute to, and use. Sponsored by DigitalOcean, a contributor can qualify for the official Hacktoberfest swag by registering and making four pull requests (PRs) between October 1-31.
+
+## What Is Hacktoberfest Initiative?
+
+During Hacktoberfest in October, we help and support our members to contribute to and/or maintain open sources.
```

Comment:
During Hacktoberfest in October, we help and support our members to contribute to and/or maintain open source repositories.